strimzi/04-service-binding.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: service.binding/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: kafka-producer-binding-user
+spec:
+  application:
+    apiVersion: app.stacks/v1beta1
+    kind: RuntimeComponent
+    name: kafka-producer
+  service:
+    apiVersion: v1beta1/kafka.strimzi.io
```

Comment:
```suggestion
    apiVersion: kafka.strimzi.io/v1beta1
```

Reply:
good catch.  doing PRs during meetings is never a good idea.  😅